src/test/scala/vsys/blockchain/state/contract/channel/ExecuteChannelContractInvalidDiffTest.scala

```diff
@@ -0,0 +1,230 @@
+package vsys.blockchain.state.contract.channel
+
+import com.google.common.primitives.{Ints, Longs, Shorts}
+import org.scalacheck.Gen
+import org.scalatest.prop.{GeneratorDrivenPropertyChecks, PropertyChecks}
+import org.scalatest.{Matchers, PropSpec}
+import vsys.account.{ContractAccount, PrivateKeyAccount}
+import vsys.account.ContractAccount.tokenIdFromBytes
+import vsys.blockchain.block.TestBlock
+import vsys.blockchain.contract._
+import vsys.blockchain.contract.channel.PaymentChannelContractGen
+import vsys.blockchain.contract.token.{SystemContractGen, TokenContractGen}
+import vsys.blockchain.state._
+import vsys.blockchain.state.diffs._
+import vsys.blockchain.transaction.contract._
+import vsys.blockchain.transaction.{GenesisTransaction, TransactionGen, TransactionStatus}
+import vsys.utils.crypto.EllipticCurveImpl
+
+class ExecuteChannelContractInvalidDiffTest extends PropSpec
+  with PropertyChecks
+  with GeneratorDrivenPropertyChecks
+  with Matchers
+  with TransactionGen
+  with TokenContractGen
+  with SystemContractGen
+  with PaymentChannelContractGen {
+
+  val preconditionsAndPaymentChannelWithAmountInvalidTest: Gen[(GenesisTransaction, GenesisTransaction, RegisterContractTransaction,
+    ExecuteContractFunctionTransaction,
+    ExecuteContractFunctionTransaction, Long, Long)] = for {
+    (master, ts, fee) <- ContractGenHelper.basicContractTestGen()
+    genesis <- genesisPaymentChannelGen(master, ts)
+    user <- accountGen
+    genesis2 <- genesisPaymentChannelGen(user, ts)
+    contract <- channelContract
+    description <- validDescStringGen
+    sysTokenId = tokenIdFromBytes(ContractAccount.systemContractId.bytes.arr, Ints.toByteArray(0)).explicitGet()
+    dataStack <- initPaymentChannelContractDataStackGen(sysTokenId.arr)
+    // Register a payment channel that supports VSYS
+    regContract <- registerPaymentChannelGen(master, contract, dataStack, description, fee, ts)
+    contractId = regContract.contractId
+    attach <- genBoundedString(2, ExecuteContractFunctionTransaction.MaxDescriptionSize)
+    depositData = Seq(master.toAddress.bytes.arr, contractId.bytes.arr, Longs.toByteArray(0L))
+    depositType = Seq(DataType.Address, DataType.ContractAccount, DataType.Amount)
+    depositVSYS <- depositVSYSGen(master, depositData, depositType, attach, fee, ts + 1)
```

Comment:
several lines of code are reused, which can use `createAndDepositVSYSPaymentChannelGen` to simplify the code.

In here `createAndDepositVSYSPaymentChannelGen` include create channel tx, we can use `_` to omit the return.

see #184 in detail